### PR TITLE
Core/DataStores: Update DBC field names to generated ones

### PR DESCRIPTION
**Pull request by Carbenium**
**Changes proposed:**

-  Core/DataStores: Update DBC field names to generated ones

**Target branch(es):** 3.3.5/master

- [x] 3.3.5
- [ ] master

**Tests performed:** (Does it build, tested in-game, etc.) compile + bit of in-game play

### CODE REVIEW

**Comment by Shauren** [comment]
Can't say I approve, master branch already established the naming convention for these generated names, using cd569e5bf28d90065d5302716806c548e7d4d305 as example, no change should be needed here, the names were already correct

**Comment by Carbenium** [comment]
Alright. I'll "diff" it against master and take names of matching fields from there.

**Comment by Carbenium** [comment]
@Shauren So, fields which are present in master as well should have now matching names. For the rest I took the generated names, but adapting them to our codestyle (removed `_` as delimiter)

File: src/server/shared/DataStores/DBCStructure.h [hunk 1]
```diff
@@ -851,386 +856,399 @@ struct HolidayNamesEntry
...
+    uint32 Date[MAX_HOLIDAY_DATES];                         // 11-36 dates in unix time starting at January, 1, 2000
+    uint32 Region;                                          // 37 WoW region
+    uint32 Looping;                                         // 38
+    uint32 CalendarFlags[MAX_HOLIDAY_FLAGS];                // 39-48
+    //uint32 HolidayNameID;                                 // 49 HolidayNames.dbc
+    //uint32 HolidayDescriptionID;                          // 50 HolidayDescriptions.dbc
+    char* TextureFilename;                                  // 51
+    uint32 Priority;                                        // 52
+    int32 CalendarFilterType;                               // 53
+    //uint32 Flags;                                         // 54 0 = Darkmoon Fair, Fishing Contest and Wotlk Launch, rest is 1
```
Comment (Shauren) [inline src/server/shared/DataStores/DBCStructure.h:870]: heh, is this some ide-integrated spellchecker? it failed, the name is "Darkmoon Faire" (HolidayNames.dbc)

File: src/server/shared/DataStores/DBCStructure.h [hunk 2]
```diff
@@ -851,386 +856,399 @@ struct HolidayNamesEntry
...
-    uint32  multimap_id;                                    // 57
-    //float BattlefieldMapIconScale;                        // 58
-    int32   entrance_map;                                   // 59 map_id of entrance map
-    float   entrance_x;                                     // 60 entrance x coordinate (if exist single entry)
-    float   entrance_y;                                     // 61 entrance y coordinate (if exist single entry)
+    uint32 ID;                                              // 0
+    //char* Directory;                                      // 1
+    uint32 InstanceType;                                    // 2
+    uint32 Flags;                                           // 3
+    //uint32 PVP;                                           // 4 0 or 1 for battlegrounds (not arenas)
```
Comment (Shauren) [inline src/server/shared/DataStores/DBCStructure.h:1068]: This field is called `MapType`

File: src/server/shared/DataStores/DBCStructure.h (hunk 1 above)
Comment (Carbenium) [reply src/server/shared/DataStores/DBCStructure.h]: Hrm, no, that was the "me"-integrated spell checker ;)

File: src/server/shared/DataStores/DBCStructure.h (hunk 2 above)
Comment (Carbenium) [reply src/server/shared/DataStores/DBCStructure.h]: Changed.

File: src/server/shared/DataStores/DBCStructure.h [hunk 3]
```diff
@@ -1280,551 +1298,551 @@ struct ScalingStatValuesEntry
...
+    int32 VehicleExitAnimBone;                              // 37
+    int32 VehicleRideAnimLoopBone;                          // 38
+    float VehicleEnterAnimDelay;                            // 39
+    float VehicleExitAnimDelay;                             // 40
+    uint32 VehicleAbilityDisplay;                           // 41
+    uint32 EnterUISoundID;                                  // 42
+    uint32 ExitUISoundID;                                   // 43
+    int32 UiSkin;                                           // 44
+    uint32 FlagsB;                                          // 45
+    //float Unk1;                                           // 46
```
Comment (Shauren) [inline src/server/shared/DataStores/DBCStructure.h:1831]: These floats are
```
    float CameraEnteringDelay;
    float CameraEnteringDuration;
    float CameraExitingDelay;
    float CameraExitingDuration;
    DBCPosition3D CameraOffset;
    float CameraPosChaseRate;
    float CameraFacingChaseRate;
    float CameraEnteringZoom;
    float CameraSeatZoomMin;
    float CameraSeatZoomMax;
```

Comment (Carbenium) [reply src/server/shared/DataStores/DBCStructure.h]: Thanks! Changed.

**Pull request merged**